datalad_gooey/app.py

```diff
@@ -295,6 +302,39 @@ def _get_info(self):
               'datalad.org</a>.'
         mbox(self.main_window, title, msg)
 
+    def _get_diagnostic_info(self):
+        self.execute_dataladcmd.emit(
+            'wtf',
+            MappingProxyType(dict(
+                result_renderer='disabled',
+                on_failure='ignore',
+                return_type='generator',
+            )),
+            MappingProxyType(dict())
+        )
+        self._cmdexec.results_received.connect(
+            self._app_cmdexec_results_handler)
```

Comment:
Move this into `__init__`, or you get an additional connection (and slowdown), each time this is triggered.

Reply:
thx!